src/components/molecules/AdminSpaceCard/AdminSpaceCard.tsx

```diff
@@ -21,28 +24,71 @@ export interface AdminSpaceCardProps {
 
 export const AdminSpaceCard: React.FC<AdminSpaceCardProps> = ({ venue }) => {
   const backgroundStyle = useCss({
-    "background-image": `url(${venue.mapBackgroundImageUrl})`,
+    background: `linear-gradient(
+        rgba(25, 24, 26, 0.8),
+        rgba(25, 24, 26, 0.8)
+      ), url(${venue.config?.landingPageConfig?.bannerImageUrl})`,
   });
   const backgroundClasses = classNames("AdminSpaceCard__bg", backgroundStyle);
-
   const logoStyle = useCss({
-    "background-image": `url(${venue.host?.icon ?? DEFAULT_VENUE_LOGO})`,
+    "background-image": `url(${venue.host?.icon || DEFAULT_VENUE_LOGO})`,
   });
   const logoClasses = classNames("AdminSpaceCard__logo", logoStyle);
 
+  const spaceIcon = SPACE_PORTALS_ICONS_MAPPING[venue.template];
+
   return (
     <div className="AdminSpaceCard">
-      <div className={backgroundClasses} />
-      <div className="AdminSpaceCard__info">
-        <div className={logoClasses} />
-        <AdminShowcaseSubTitle>{venue.name}</AdminShowcaseSubTitle>
+      <div className={backgroundClasses}>
+        <div className="AdminSpaceCard__bg-container">
+          <Link className="AdminSpaceCard__link" to="">
+            {"Visit"}
+            <FontAwesomeIcon
+              className="AdminSpaceCard__link-icon"
+              icon={faExternalLinkAlt}
+            />
+          </Link>
+          <div className="AdminSpaceCard__body">
+            <div className={logoClasses} />
+            <div className="AdminSpaceCard__body-info">
+              <AdminShowcaseSubTitle>{venue.name}</AdminShowcaseSubTitle>
```

Comment:
Maybe not the proper component for this place. Can't hurt to make a copy of it, name it `CardSubTitle`, add the proper SCSS there, and never need bother overriding the `AdminShowcaseSubTitle` style

Reply:
Well, considering I don't see `AdminShowcaseTitle` here... can be named `CardTitle` instead 🤷

Reply:
yeah, you're right
did that f9043d4